src/client/order-components/GoogleAPI.js

```diff
@@ -18,14 +23,39 @@ const formatAnswer = (mode, value) => {
   return winnerEstimate;
 };
 
-export const initMap = () => {
+// hacky-global scope variables to manage state for the requestRide
+const rideETA = {};
+const rideFare = {};
+
+export const requestRide = (mode) => {
+  let url = `http://localhost:8000/internal/requestRide`;
```

Comment:
created a trello card for this - https://trello.com/c/tnkEnQMS/137-env-node-react not as straight forward as in the server code